Refs #86 Basic TextInput

narek-h opened the pull request:
Added basic TextInput component with basic properties and events support.
Added RNTester example showing supported functionality. 

Note: Sometimes noticing a big delay on updating a text view for the occurred event.  Continue to type/select to see the updates. 
(probably the issue is not relevant to this patch)

MaxRis commented on RNTester/js/TextInputExample.js:
```diff
@@ -0,0 +1,258 @@
+/**
```
It might make more sense to name file TextInputExample.ubuntu.js since TextInput implementation for desktop differs from implementation for mobile.
Also copyright might be replaced for Status.

narek-h commented:
@maxr, thanks for the comments. Will add a patch addressing them,  
Also, will add unit tests by a separate PR.

narek-h commented:
@MaxRis, @vkjr , added  `js/utils.js` file containing the function used in qml. Later we can rename/split them to more specific files if needed. 
Addressed the other comments as well.  If everything looks ok, will merge.

MaxRis commented:
@narek-h looks great! Thank you!
